apps/blockchain/lib/blockchain/account/storage.ex

```diff
@@ -9,35 +9,30 @@ defmodule Blockchain.Account.Storage do
   """
 
   alias ExthCrypto.Hash.Keccak
-  alias MerklePatriciaTree.{DB, Trie}
+  alias MerklePatriciaTree.TrieStorage
```

Comment:
Could these two lines be `alias MerklePatriciaTree.{TrieStorage, Trie}`? This could also be a case where we can remove the `Trie` alias and force account/storage to interact through `TrieStorage` if it makes sense.